boot/menhir/menhirLib.ml

```diff
@@ -3513,5 +3513,5 @@ module MakeEngineTable (T : TableFormat.TABLES) = struct
 end
 end
 module StaticVersion = struct
-let require_20181113 = ()
+let require_20190626 = ()
```

Comment:
Is that version actually required? Or can we stick to an older version of menhir?

I guess the question I want to ask is: do we want to make sure we keep the oldest possible version there, or do we not care at all because it only affects compiler devs anyway?

Reply:
I would hesitate between "we don't care" and "we should only accept increases to this field to avoid back-and-forth; it's easy for users of older versions to upgrade anyway".